api/schemas/serializers.py

```diff
@@ -57,7 +52,10 @@ def get_absolute_url(self, obj):
     class Meta:
         type_ = 'form_blocks'
 
+
 class RegistrationSchemaSerializer(SchemaSerializer):
+    description = ser.CharField(read_only=True, allow_blank=True)
```

Comment:
high-level description for the schema displayed here - was originally on config.tooltip in earlier version of this work.